src/generic_ui/polymer/description.ts

```diff
@@ -3,10 +3,11 @@
 
 declare var core :uProxy.CoreAPI;
 
+console.log('loading description ' + model.globalSettings.description);
 Polymer({
-  description: model.globalSettings.description,
+  model : model,
   update: function() {
-    model.globalSettings.description = this.description;
+    model.globalSettings.description = this.model.globalSettings.description;
```

Comment:
Is this line really necessary?  Since this.model == model, won't this be a no-op?